apps/studio/components/ui/AIAssistantPanel/AssistantChatForm.tsx

```diff
@@ -37,6 +37,10 @@ export interface FormProps {
   snippetsClassName?: string
   /* Additional class name for the form wrapper */
   className?: string
+  /* Editing mode props */
+  isEditing?: boolean
+  editingMessageContent?: string
```

Comment:
doesn't look we use editingMessageContent here

Reply:
Thanks I fixed it! [`290db4d` (#37603)](https://github.com/supabase/supabase/pull/37603/commits/290db4de2043f88d9b1597d2ca8ce61a22e19611)